packages/core/theme/src/components/popover.ts

```diff
@@ -36,20 +36,20 @@ const popover = tv({
       // visibility
       "data-[arrow=true]:before:block",
       // top
-      "data-[placement=top]:before:-bottom-1",
+      "data-[placement=top]:before:-bottom-[calc(theme(spacing.1)_-_1.5px)]",
```

Comment:
Why the `-1.5px`?

Reply:
This makes the arrow look better, try removing the 1.5 by 1px